mobile-ui/src/main/res/values/ids.xml

```diff
@@ -31,19 +31,65 @@
   -->
 
 <resources>
-    <id name="title" />
-    <id name="description" />
-
+    <!-- Navigation Action Bar -->
     <id name="rs2_step_navigation_action_forward" />
     <id name="rs2_step_navigation_action_backward" />
     <id name="rs2_step_navigation_action_skip" />
+    <id name="rs2_step_navigation_shadow"/>
+    <!-- Step Header -->
+    <id name="rs2_step_header_cancel_button"/>
+    <id name="rs2_step_header_info_button"/>
+    <id name="rs2_step_header_progress_label"/>
+    <id name="rs2_step_header_progress_bar"/>
+    <id name="rs2_step_header_progress_bar_floating"/>
+    <id name="rs2_step_header_action_button_group"/>
+
+    <id name="title" />
+    <id name="description" />
+
     <id name="rs2_step_navigation_action_cancel" />
     <id name="rs2_step_navigation_action_learn_more" />
     <id name="rs2_step_navigation_action_add_more" />
 
+    <id name="footer_action_forward"/>
+    <id name="footer_action_backward"/>
+    <id name="footer_action_reminder"/>
+
     <id name="rs2_step_navigation_action_bar"/>
 
     <id name="rs2_step_container"/>
     <id name="rs2_current_step" />
     <id name="rs2_step_id" />
+
+    <!-- Id's for NavigationViewBinding -->
+    <id name="navigationHeaderView"/>
+    <id name="navigationFooterView"/>
+    <id name="navigationBodyView"/>
+    <id name="titleLabel"/>
```

Comment:
I usually see "label" used when there are multiple associated fields, e.g. 
```
TextView addressLabel; 
ImageView addressIcon;
TextView addressInput;
```

I'd suggest rs2_step_text, res2_step_title, etc

Reply:
Same as the above comment these bindings are about to change substantially so I would prefer to make the change there if that is ok.